### PR TITLE
feat: fabric operations console

### PR DESCRIPTION
**Pull request by czar0**
[Fabric Operations Console](https://github.com/hyperledger-labs/fabric-operations-console) with all its incredible features (see 👇 ) is finally here! 🥳 

This PR includes the following:
- Support for all the features offered off-the-shelf by the console
- One-command run - `fabkit console start`
- Automatic creation of components artefacts starting from current network config (multi-org! 👯 )
- Step-by-step [guide](https://github.com/everledger/fabkit/blob/4a081baee2f6a1a7a2d06bacb0ed8c8c23ac2bb9/docs/console.md) to one-time configuration and getting started

> 
> The console provides the following high level function:
> 
> - Ability to import and manage all Hyperledger Fabric Components from a single web console, no matter where they are located.
> - Maintain complete control over identities, channels, and smart contracts.
> - Join Peers to Channels and view channel membership as well as individual transactions and channel details.
> - Register, view, delete, and re-enroll CA Users.
> - View Ordering cluster and node information as well as view and modify consortium and channel membership.
> - View and modify channel capabilities and ordering service parameters.
> - Install and Instantiate chaincode.  Supports both 1.x and 2.x Lifecycle.
> - View, Create, Import and Export Organizations and Identities.
> - Role Based Access Control in UI to tightly control which Console users can perform which operations.

### CODE REVIEW

**Comment by gtpaulose** [comment]
https://github.com/everledger/fabkit/blob/4a081baee2f6a1a7a2d06bacb0ed8c8c23ac2bb9/fabkit#L26 - this line makes it hard for people to run fabkit when they don't have or have poor internet.

Can you add a TODO to implement a caching mechanism for this?

**Comment by czar0** [comment]
> https://github.com/everledger/fabkit/blob/4a081baee2f6a1a7a2d06bacb0ed8c8c23ac2bb9/fabkit#L26
> 
> - this line makes it hard for people to run fabkit when they don't have or have poor internet.
> Can you add a TODO to implement a caching mechanism for this?

I thought about it in the past and then opted for this on-the-fly approach after realising that a peer will always require an internet connection in order to deploy a chaincode container.

**Comment by gtpaulose** [comment]
I removed the check dep version from the main script because it was unnecessarily increasing the overhead time. We don't need to check everytime we run Fabkit whether they have the required dependencies.

**Review by gtpaulose — APPROVED** [review]

File: scripts/console.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+start_console() {
+    loginfoln "Starting console"
+
+    if ! docker ps | grep -q "fabric"; then
```
[the remark this answers is not in the document]
Comment (czar0) [reply scripts/console.sh:6]: Good suggestion. Let's add this as an enhancement proposal in a new ticket.

**Comment by czar0** [comment]
@gtpaulose I reverted one commit which was removing the dependency check. While I think it is a good suggestion, I would rather prefer to do this in another ticket (and perhaps architecting a good caching mechanism). Moreover, the `__check_dep_version` would have been reachable from the `install.sh` script when run on the fly.

**Pull request merged**